browser/about_flags.cc

```diff
@@ -169,6 +171,11 @@ constexpr char kBraveDebounceDescription[] =
 constexpr char kBraveDeAMPName[] = "Enable De-AMP";
 constexpr char kBraveDeAMPDescription[] = "Enable De-AMPing feature";
 
+constexpr char kBraveGoogleSignInName[] =
+    "Enable Google Sign-In Permission Prompt";
+constexpr char kBraveGoogleSignInDescription[] =
+    "Enable permissioning access to legacy Google Sign-In";
```

Comment:
why `legacy`? google has something new that doesn't require this permission?

Reply:
Yup, I just encountered this as well in the security issue. Here's the docs for the recommended method to use google login services in the future: https://developers.google.com/identity/gsi/web/guides/overview

Reply:
Exactly, AFAIK, the new recommended method by Google doesn't use 3p cookies, and hence is not a privacy leakage. With this PR I want to be able to remove the global 3p cookie exception for `https://accounts.google.com/o/oauth2/auth*` we currently need for some websites.